src/spam-protection/get-google-recaptcha-token.ts

```diff
@@ -0,0 +1,15 @@
+import { ReadableDataStore } from '@bigcommerce/data-store';
+
+import { InternalCheckoutSelectors } from '../checkout';
+
+import CardingProtectionActionCreator from './carding-protection-action-creator';
+
+export default function getGoogleRecapthaToken(store: ReadableDataStore<InternalCheckoutSelectors>, error: any) {
+    const { additional_action_required, status } = error;
```

Comment:
`error` might not be in the shape of an object that you expect so you should check it before using it. If you get rid of the error response parsing logic in `PaymentActionCreator`, here you should be expecting an `Action` object with `payload` property that contains the API response.